### ISSUE TITLE
_warnings settings are process-wide

### ISSUE THREAD

**Issue by pitrou** [python/cpython#58027]
BPO | [13819](https://bugs.python.org/issue13819)
--- | :---
Nosy | @pitrou, @alex, @nanjekyejoannah

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2012-01-18.20:50:16.644>
labels = ['extension-modules', 'interpreter-core', 'type-bug']
title = '_warnings settings are process-wide'
updated_at = <Date 2019-07-18.17:47:39.164>
user = 'https://github.com/pitrou'
```

bugs.python.org fields:
```python
activity = <Date 2019-07-18.17:47:39.164>
actor = 'nanjekyejoannah'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Extension Modules', 'Interpreter Core']
creation = <Date 2012-01-18.20:50:16.644>
creator = 'pitrou'
dependencies = []
files = []
hgrepos = []
issue_num = 13819
keywords = []
message_count = 2.0
messages = ['151563', '223359']
nosy_count = 3.0
nosy_names = ['pitrou', 'alex', 'nanjekyejoannah']
pr_nums = []
priority = 'low'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue13819'
versions = ['Python 2.7', 'Python 3.4', 'Python 3.5']
```

</p></details>

**Comment by iritkatriel** [comment]
CC @ericsnowcurrently

**Comment by aisk** [comment]
Hi, I see there is another similar issue in #80918, which was already solved by #13159, so I guess this can be closed now?
